app/src/main/java/openfoodfacts/github/scrachx/openfood/features/shared/views/TipBox.kt

```diff
@@ -19,7 +23,7 @@ class TipBox(context: Context, attrs: AttributeSet?) : LinearLayout(context, att
 
     private val prefs: SharedPreferences by lazy { PreferenceManager.getDefaultSharedPreferences(getContext()) }
 
-    private val binding = TipBoxBinding.inflate(LayoutInflater.from(context), this, false)
+    private val binding = TipBoxBinding.inflate(LayoutInflater.from(context), this, true)
```

Comment:
Why this?

Reply:
In general cases (lets say Fragments) , FragmentManager is responsible for adding childView. In this case, the layout file does not have a root ViewGroup , so we specify our custom LinearLayout to be its root and hence attach to root true. You can refer the following :- https://www.bignerdranch.com/blog/understanding-androids-layoutinflater-inflate/

This was the reason the tip system was broken(not working).